evap/staff/views.py

```diff
@@ -285,6 +289,28 @@ def semester_import(request, semester_id):
 
 
 @staff_required
+def semester_export(request, semester_id):
+    semester = get_object_or_404(Semester, id=semester_id)
+
+    ExportSheetFormset = formset_factory(form=ExportSheetForm, can_delete=True, extra=1)
+    formset = ExportSheetFormset(request.POST or None, form_kwargs={'semester': semester})
+
+    if formset.is_valid():
+        ignore_not_enough_answers = request.POST.get('ignore_not_enough_answers', False)
```

Comment:
if it's checked it might be something like `on` or `selected`. but all non-empty strings are true.